### ISSUE TITLE
JS Identifiers exercise needs better test coverage

### ISSUE THREAD

**Issue by mikesjewett** [Bloc/workshop-curriculum#46]
https://workshop.bloc.io/exercises/identifiers

Currently, the tests can all show as passing, even if the exercise isn't correct. This results in all greens, but a failure message in the results area:

![image](https://cloud.githubusercontent.com/assets/1168076/17109196/f13741a6-5264-11e6-9284-e128f0dfbd92.png)

The example case for this was that a user's solution was this:

```javascript
var movieTitle = "Star Wars";
var The_Sith_Lord;
var spaceship = "Death Star";
var space_Gangster;
var hero = "Skywalker";
var PRINCE;
```

Note that they changed `PRINCE$$` to `PRINCE`, but didn't receive a failure message.

**Comment by mikesjewett** [comment]
@philspitler wondering if you'd have time to look at this since Colleen is out?

**Comment by mmaxwell** [comment]
They actually _are_ receiving the failure message, but it is below the fold.  This test is operating exactly how you'd expect it to, but there are tests that are below the three that pass.  You can see that the `PRINCE$$` test does actually fail when you name it `PRINCE`.  The correct class, `.bg-danger` is being applied, but you are unable to see it or scroll to it.

![image](https://cloud.githubusercontent.com/assets/2146353/17114425/833ea666-5274-11e6-93c1-6134ce9564c2.png)

You can see that the test does return with this as failed:

![image](https://cloud.githubusercontent.com/assets/2146353/17114445/96e23ff2-5274-11e6-90ca-8fbd85424089.png)

I think this is an issue with the workshop tool itself, and not with these specific tests.

**Comment by mmaxwell** [comment]
In the inspector, if I right click on the containing `p` and click "Scroll into view", you will see that the test results window will scroll down to the failed test: 

![image](https://cloud.githubusercontent.com/assets/2146353/17114496/c9f5681a-5274-11e6-81a4-ecf35b133827.png)

**Comment by mmaxwell** [comment]
If I change `.editor .resultsWrapper` margin to be `-400px`, which matches the height of `.editor .results`, everything appears normally on the page:

![image](https://cloud.githubusercontent.com/assets/2146353/17114680/92e19f1e-5275-11e6-8a05-3354c93805f9.png)

**Comment by mikesjewett** [comment]
Thanks @mmaxwell closing this since it's a platform bug and will be addressed by bloc/workshop#489
